parts/kubernetesmasterresources.t

```diff
@@ -395,6 +396,7 @@
           {{end}}
         },
         "storageProfile": {
+          {{GetEtcdDisk}}
```

Comment:
Do we really need a new template function for this? :( Is this because it needs information that's not available in the model passed to the template here? Looks like maybe the OrchType or MasterProfile?